guides/github.md

```diff
@@ -0,0 +1,84 @@
+# Guidelines for GitHub
+
+The following guidelines have agreed upon in the backend team to formalize how we work with GitHub, and the expectations we have between us as developers.
+
+A quick summary of the guidelines, followed by sections with more information for each point:
+
+**Draft PRs**
+1. Draft PRs can and should be used when you want to run the GitHub checks on your branch, receive review from specific people, or to easily share and discuss code.
+2. Draft PRs should not be reviewed unless it has been requested by the author.
+3. Draft PRs should be used over normal PRs marked "WIP", to avoid premature or accidental merging of incomplete code.
+4. Draft PRs can be left for longer periods of time, but after 1 month of no change, action should be taken by the author
+
+**PR life / age**
+1. PRs should receive some feedback within 2 days, and within 5 days it should have 2 reviews. Exceptions for bigger or complicated PRs that require more effort to review.
+2. The author is responsible to resolve any comment, fix any failing checks and requesting reviewers as needed.
+3. The author is responsible for the PR from its creation, to its resolution, be it either merging or closing the PR.
+
+**Reviews**
+1. PRs against master should always have a minimum of 2 approvals.
+2. PRs against previous versions only require 1 approval, but the author should require more approvals if a backport contains significantly different code than the original PR against master.
+
+**Merging PRs**
+1. The author is primarily responsible for merging their own PRs
+2. Merging should only happen when all comments have been resolved
+3. Auto-merging PRs should be used when the author is confident no further work is required
+4. Other developers may only merge PRs they have not authored when all checks are OK, two approvals exist and no comment lies unresolved.
+
+
+**GitHub conventions**
+1. PR titles must follow the naming convention, prepending the title with "fix|feat|chore|test|ci|", based on the content of their PR.
+2. All PRs (omitting Draft PRs) must have a description explaining the content of the PR.
+3. All branches should be named on the format: "<Jira-issue-number>[_<version>]". Example DHIS2-123 and DHIS2-123_2.40.
```

Comment:
I found a small issue with the pattern using version as suffix. 
If you have multiple PRs for same issue number you have to add another suffix, like `DHIS2-1234-additional`. If you then backport this if is unclear what to do. `DHIS2-1234-2.39-additional`?
Therefore I suggest making the version a prefix: `2.39_DHIS2-1234-additional`. That way it is always `<version>_<unversioned-branch-name>`.